cicecore/cicedynB/analysis/ice_history_bgc.F90

```diff
@@ -226,20 +226,20 @@ module ice_history_bgc
            n_upNO        , n_upNH        , & 
            n_bTin        , n_bphi        , &
            n_iDi         , n_iki         , &
-           n_bgc_PON     , n_bgc_PON_ml  , &
+           n_bgc_PON     ,                 &
```

Comment:
I assume these BGC variables are never used? I had left a bunch in as they were potentially to be used in the future.